BeeSwift/AppDelegate.swift

```diff
@@ -140,19 +137,16 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
     // MARK: - UNUserNotificationCenterDelegate
     
     func userNotificationCenter(_ center: UNUserNotificationCenter, willPresent notification: UNNotification, withCompletionHandler completionHandler: @escaping (UNNotificationPresentationOptions) -> Void) {
-        switch notification.request.identifier {
-        case JSONGoal.unlockNotificationIdentifier:
-            // about to present a notification, a reminder to unlock the device
-            // yet the app is active, thus we can abandon the notification
-            completionHandler([])
-        default:
-            completionHandler([.alert, .sound, .badge])
-        }
+        completionHandler([.alert, .sound, .badge])
     }
     
     private func resetStateIfUITesting() {
         if ProcessInfo.processInfo.arguments.contains("UI-Testing") {
             UserDefaults.standard.removePersistentDomain(forName: Bundle.main.bundleIdentifier!)
         }
     }
+    
+    private func removeAllLocalNotifications() {
+        UNUserNotificationCenter.current().removeAllPendingNotificationRequests()
```

Comment:
hm, possibly this may have been enough for those opening the app regularly. Queued notification requests would have been removed upon app start. 

The lines removed in L143-L150 on the other hand saw to it that the "unlock your phone" message did not appear while the app was in the foreground.